.github/workflows/agw-coverage.yml

```diff
@@ -0,0 +1,232 @@
+        with:
+          image: ${{ env.BAZEL_BASE_IMAGE }}
+          # TODO: Remove work-around mount of Github workspace to /magma (https://github.com/addnab/docker-run-action/issues/11)
+          options: -v ${{ github.workspace }}:/workspaces/magma/ -v ${{ github.workspace }}/lte/gateway/configs:/etc/magma
+          run: |
+            cd $MAGMA_ROOT
+            bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION_C_CPP }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}"
+            # Collecting coverage with Bazel can be slow. We can follow this thread to see if this can be improved: https://github.com/bazelbuild/bazel/issues/8178
+            # Coverage in bazel is flaky for remote caches - the flags below are helping. See GH13026 for details.
+            bazel coverage \
+              --profile=Bazel_test_cc_coverage_profile \
+              --experimental_split_coverage_postprocessing --experimental_fetch_all_coverage_outputs --remote_download_outputs=all \
+              //orc8r/gateway/c/...:* //lte/gateway/c/...:*
+            # copy out coverage information into magma so that it's accessible from the CI node
+            cp bazel-out/_coverage/_coverage_report.dat $MAGMA_ROOT
+      - name: Upload code coverage
+        if: always()
+        id: c-cpp-codecov-upload
+        uses: codecov/codecov-action@d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70 # pin@v3.1.1
+        with:
+          flags: c_cpp
+      - name: Publish bazel profile
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # pin@v3
+        if: |
+          always() &&
+          github.repository_owner == 'magma' &&
+          github.ref_name == 'master'
+        with:
+          name: Bazel test C/C++ coverage profile
+          path: Bazel_test_cc_coverage_profile
+      - name: Extract commit title
+        # yamllint enable
+        if: failure() && github.event_name == 'push'
+        id: commit
+        run: |
+          str="$(jq '.head_commit.message' $GITHUB_EVENT_PATH)"    # get the head_commit message
+          echo ::set-output name=title::${str%%\\n*} | tr -d '"'
+      - name: Build space left after run
+        shell: bash
+        run: |
+          echo "Available storage:"
+          df -h
+      - name: Notify Bazel C/C++ coverage failure to slack
+        if: failure() && steps.bazel-cc-codecoverage.conclusion == 'failure' && github.event_name == 'push'
+        uses: rtCamp/action-slack-notify@12e36fc18b0689399306c2e0b3e0f2978b7f1ee7 # pin@v2.2.0
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_BAZEL_CI }}
+          SLACK_TITLE: "C / C++ code coverage with Bazel"
+          SLACK_USERNAME: "${{ github.workflow }}"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+          SLACK_FOOTER: ' '
+          MSG_MINIMAL: actions url,commit
+      - name: Notify C/C++ coverage upload failure to slack
```

Comment:
Should we combine the slack notifications (same below)?
* send only to bazel channel
* ~ `(steps.bazel-cc-codecoverage.conclusion == 'failure' || steps.c-cpp-codecov-upload.outcome=='failure') && github.event_name == 'push'`

Reply:
Since this is a de-facto switchover of the C/CPP/Python coverage, would it not make more sense to only send it to the standard CI channel? Else we send it to the bazel channel now and then we migrate all channels to the standard CI channel during the final switchover?

Reply:
I prefer the "else" option - send it to the bazel channel for now and switch this to #ci later.

Reply:
Done